src/components/flowchart/components/slice-pipeline-action/slice-pipeline-action.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Button from '../../../ui/button';
+
+import './slice-pipeline-action.scss';
+
+export const SlicePipelineAction = ({ chartSize, filteredPipeline }) => {
```

Comment:
The name ix a bit confusing because it made me think it was an action rather than a component. Perhaps we could rename it to something like PipelineSliceIndicator ?

Reply:
hey i agree the name is a bit confusing, especially later on it shows the run command too. How about `FilteredPipelineActionBar` ? It's following what Steps used in Figma, and it's also consistent with selectors `FilteredPipeline`